sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprContainerIT.java

```diff
@@ -76,6 +76,9 @@ public void setDaprProperties() {
   }
 
   private void configStub() {
```

Comment:
Shouldn't we add the `withAppHealthCheckPath(...)` to `DaprContainer` using `/actuator/health`?

Reply:
added